packages/gafl-webapp-service/README.md

```diff
@@ -17,6 +17,7 @@ To run from this directory:
 | REDIS_PASSWORD             | Password used to authenticate with the configured redis instance |    no    |                     |                               |
 | CHANNEL                    | The sales channel                                                |    no    | websales            | websales, telesales           |
 | SESSION_COOKIE_NAME        | Name of the session cookie                                       |    no    | sid                 |                               |
+| SESSION_COOKIE_PASSWORD    | Encryption key for the session cookie                            |   yes    |                     |                               |
```

Comment:
Perhaps include the detail about it needing to be a 32-character string

Reply:
ok

Reply:
Re-reviewed and this has been actioned